pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+# TODO: update name (cromshell) and version (2.0.0.beta.1)
```

Comment:
what do we do with this todo?  is this handled by the sed lines in github actions?